cli/tests/testdata/main-values.yaml

```diff
@@ -31,13 +31,20 @@ timescaledb-single:
       size: 150Gi
     wal:
       size: 20Gi
+  patroni:
+    bootstrap:
+      dcs:
+        postgresql:
+          parameters:
+            max_connections: 300
 
 # Values for configuring the deployment of the Promscale Connector
 # The charts README is at:
 #   https://github.com/timescale/promscale/tree/master/helm-chart
 promscale:
   enabled: true
-  image: timescale/promscale:0.3.0
+  image: vineeth97/promscale:backfill-latest
```

Comment:
Once the `0.4.0` Promscale is released we need to update to that image. Currently to test HA setup using the latest Promscale master branch image.

Reply:
The release is now out. Please remember to switch this back